src/amo/pages/Guides/index.js

```diff
@@ -135,6 +148,9 @@ export class GuidesBase extends React.Component<InternalProps> {
 
           <div
             className="Guides-section-explore-more"
+            onClick={(e) => this.handleClick(e, url)}
+            onKeyDown={(e) => this.handleClick(e, url)}
+            role="link"
```

Comment:
note: I wasn't totally sure about the onKeyDown and the role here but linting was complaining and I believe these seemed to make the most sense.